backend/apps/predictions/views.py

```
@@ -0,0 +1,255 @@
+                update_reviewer_availability()
+                msg = "Monitoring task executed synchronously."
+
+        return Response({
+            "status": msg,
+            "details": details,
+        }, status=status.HTTP_200_OK)
+
+
+
+class TrainModelAPIView(views.APIView):
+    permission_classes = [AllowAny]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)